tests/results/scale/edge/edge-oss.md

```diff
@@ -135,28 +135,28 @@ The logs are attached only if there are errors.
 
 ### Reloads
 
-- Total: 98
+- Total: 60
```

Comment:
This is interesting @sjberman. That's a pretty significant decrease in the # of reloads.

Reply:
Yeah....wonder why that is

Reply:
Same. We haven't committed any changes that should affect the reload/batch processing performance. Maybe something to do with the k8s api?

Reply:
I don't think we've committed anything related to this. Could also be a fluke with the metrics reporting.

Reply:
> Could also be a fluke with the metrics reporting.

That would be a bummer.